packages/react-server/src/ReactFizzServer.js

```diff
@@ -33,6 +33,7 @@ import type {ComponentStackNode} from './ReactFizzComponentStack';
 import type {TreeContext} from './ReactFizzTreeContext';
 import type {ThenableState} from './ReactFizzThenable';
 import {enableRenderableContext} from 'shared/ReactFeatureFlags';
+import {describeObjectForErrorMessage} from 'shared/ReactSerializationErrors';
```

Comment:
Fizz is typically not a "serializer", except in this one case where we serialize errors it is.